indico/modules/events/contributions/templates/management/_contribution_list.html

```diff
@@ -112,12 +124,66 @@
                                    data-confirm="{% trans title=contrib.title %}Are you sure you want to completely delete contribution '{{ title }}'?{% endtrans %}"
                                    data-update="#contribution-list"
                                    data-method="DELETE"
-                                   data-href="{{ url_for('.manage_delete_contrib', contrib) }}"></a>
+                                   data-href="{{ url_for('.manage_rest_contrib', contrib) }}"></a>
                             </td>
                         </tr>
                     {% endfor %}
                 </tbody>
             </table>
         </form>
+        <script>
+            (function() {
+                'use strict';
+
+                var sessionPickerOptions = {
+                    containerClasses: 'session-item-container',
+                    actionButtonTitle: $T.gettext('Add new session'),
+                    createItem: function(itemData) {
+                        var item = $('<div>', {'class': 'dropdown-item', 'data': {'filter': itemData.title},
+                                               'title': itemData.title});
+                        item.append($('<span>', {'class': 'session-bgcolor-preview',
+                                                 'css': {'background': '#' + itemData.colors.background}}));
+                        item.append($('<span>', {'class': 'session-title', 'text': itemData.title}));
+                        return item;
+                    },
+                    onSelect: function(selectedItem) {
+                        var $this = $(this),
+                            styleObject = $this[0].style;
+
+                        $.ajax({
+                            url: $this.data('href'),
+                            method: $this.data('method'),
+                            data: JSON.stringify({session_id: selectedItem.id}),
+                            dataType: 'json',
+                            contentType: 'application/json',
+                            error: handleAjaxError,
+                            complete: IndicoUI.Dialogs.Util.progress()
+                        });
+
+                        $this.find('.label').text(selectedItem.title);
+                        styleObject.setProperty('color', '#' + selectedItem.colors.text, 'important');
+                        styleObject.setProperty('background', '#' + selectedItem.colors.background, 'important');
+                    }
+                };
+
+                $('.session-item-picker')
+                    .itempicker(sessionPickerOptions)
+                    .on('indico:actionButtonClicked', function() {
+                        ajaxDialog({
+                            title: $T.gettext('Add new session'),
+                            url: {{ url_for('sessions.create_session', event) | tojson }},
+                            confirmCloseUnsaved: true,
```

Comment:
useful or overkill? it's not like there's much to write like it's the case for notes/minutes...

ping @pferreir